utils/stm.stanza

```diff
@@ -6,58 +7,72 @@ defpackage ocdb/stm:
   import reader
   import jitx
 
+defstruct StmError <: Exception : 
+  message:?
+
+defmethod print (o:OutputStream, e:StmError) : 
+  print(o, "Could not extract STM component. %_." % [message(e)])
+
+defstruct MultiplePinsMappedToSamePad <: Exception : 
+  multi-pins: Tuple<KeyValue<String|Int, Seqable<String|Int>>>
+
+defmethod print (o:OutputStream, e:MultiplePinsMappedToSamePad) :
+  print(o, "Multiple pins are mapped to the same pad:")
+  for mp in multi-pins(e) do : 
+    lnprint(IndentedStream(o), "pad %_: %," % [key(mp), value(mp)])
+
 ;=================================================
 ;========== STM JSON -> Stanza Structs ===========
 ;=================================================
 
 ;JITX Pin Properties to be created in JITX
-public defstruct STMPinProperties :
+public pcb-struct ocdb/stm/STMPinProperties :
   generic-pin: STMGenericPin
   power-pin: STMPowerPin
   rows: Tuple<STMPinProperties-Row>
 
-public defstruct STMGenericPin :
+public pcb-struct  ocdb/stm/STMGenericPin :
```

Comment:
you have two spaces instead of one between `pcb-struct` and `ocdb/...`

Reply:
Fixed!